csbuild/__init__.py

```diff
@@ -563,7 +563,7 @@ class Scope(ContextManager):
 		"""
 		def __init__(self, *scopeTypes):
 			for scopeType in scopeTypes:
-				assert scopeType == Csbuild.ScopeDef.Intermediate or scopeType == Csbuild.ScopeDef.Final, "Invalid scope type"
+				assert scopeType in (Csbuild.ScopeDef.Intermediate, Csbuild.ScopeDef.Final), "Invalid scope type"
```

Comment:
Was this really a pylint error? This seems like it would be less efficient since it has to create the tuple and iterate through it and then destroy it...

Reply:
Yes, it was giving an error about it.  My bet is they're going for cleaner code?  But, I have no way of knowing for certain.  All I know is it was flagging a new pylint error and I agree the new way does look much better.

Reply:
Sure, the new way looks better, but where are we drawing the line on cleanliness vs efficiency?

Reply:
In this case, the performance impact will be minimal and likely negligible, so I'm opting for readability.  However, I will disable the pylint check on this in the pylintrc since there are valid cases for doing it both ways.